test/js/css-translation.js

```diff
@@ -83,4 +83,92 @@ suite('CSSTranslation', function() {
     var expectedMatrix = new DOMMatrixReadonly([1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 3, 0.5, -4, 1]);
     typedOM.internal.testing.matricesApproxEqual(translation.matrix, expectedMatrix);
   });
+
+  test('Parsing valid basic strings results in CSSTranslation with correct values', function() {
+    var values = [
+      {str: 'translate(10PX)', x: 10, y: 0},
+      {str: 'translate(-13px)', x: -13, y: 0},
+      {str: 'TrAnSlAtE(14px)', x: 14, y: 0},
+      {str: 'translate(11px, 12px)', x: 11, y: 12},
+      {str: 'translate(-13px, -14px)', x: -13, y: -14},
+      {str: 'TrAnSlAtE(15px, 16px)', x: 15, y: 16},
+    ];
+    for (var i = 0; i < values.length; i++) {
+      var parsed = typedOM.internal.parsing.consumeTranslation(values[i].str);
+      assert.isNotNull(parsed, values[i].str + ' should parse a CSSTranslation');
+      assert.strictEqual(parsed[1], '', values[i].str + ' should not return any trailing characters after consume');
```

Comment:
It would be good to test that the trailing characters is reported correctly in at least one of the tests. In some of my later patches I've added an extra field to the objects in `values` and `strictEqual`'d it here instead of always comparing with `''`